### DISCUSSION TITLE
DNS chceck unsuccesful

### DISCUSSION THREAD

**Discussion by oliwierczaplicki** [community/community#55025] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello! 
I'am writing this post because i need help with my custom domain. 
I searched everything i probably can i'm new to this stuff and don't have idea what to do.
So basically i want to set my own domain on github pages and aws route53 everything was working 'till I change something in my repo 
and then my page stopped working. I tried to do the same steps from this tutorial: [https://medium.com/@benwiz/how-to-deploy-github-pages-with-aws-route-53-registered-custom-domain-and-force-https-bbea801e5ea3] but whenever i want to add my custom domain it says exactly whats on screenshot 
![obraz_2023-05-10_110600246](https://github.com/community/community/assets/131268103/dbd26945-7e66-4b55-aaaa-f1bc45353180)

**Comment by airtower-luna** [comment]
When did you set up the domain? With DNS caching it can take a while until the GitHub servers see the new record. The A records look as [described in the official documentation](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain) to me, the AAAA records seem to be missing.
